encryptedlogging/src/main/kotlin/com/automattic/encryptedlogging/utils/Utils.kt

```diff
@@ -0,0 +1,17 @@
+package com.automattic.encryptedlogging.utils
+
+import java.util.Locale
+
+internal object Utils {
+    internal fun Int.toMB() = String.format(
+        Locale.US,
+        "%.2f",
+        this.toFloat() / 1024.0 / 1024.0
+    )
+
+    internal fun Long.toMB() = String.format(
+        Locale.US,
+        "%.2f",
+        this.toFloat() / 1024.0 / 1024.0
+    )
+}
```

Comment:
nitpick: if `object` is internal, I believe its methods don't have to

```suggestion
internal object Utils {
    fun Int.toMB() = String.format(
        Locale.US,
        "%.2f",
        this.toFloat() / 1024.0 / 1024.0
    )

    fun Long.toMB() = String.format(
        Locale.US,
        "%.2f",
        this.toFloat() / 1024.0 / 1024.0
    )
}
```

Reply:
Just committed the suggestion @wzieba , thanks: ae25826f9152c580be9429a5546dbc065fa41f30